index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio Project</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+   <header>
+    <nav class="navbar">
+        <div class="logo-div">
+            <p class="logo">My Logo</p>
+            <img src="images/Union.png" alt="image" class="nav-image">
+        </div>
+        <div class="navbar-items">
+            <ul>
+                <li><a href="#" class="portfolio">Portfolio</a></li>
+                <li><a href="#" class="about">About</a></li>
+                <li><a href="#" class="contact">Contact</a></li>
+                </ul>
+        </div>
+    </nav>
+</header> 
```

Comment:
🥳 Kudos for creating an excellent `header` with his corresponding links on the `class=navbar-items`

- [ ] As in this mobile version, you will have a logo at the top left of the screen. Please add `a` tag to "My Logo" with a link to the first section of the page.